code/ui/world/player/NameTag.razor

```diff
@@ -0,0 +1,39 @@
+﻿@using Sandbox;
+@using Sandbox.UI;
+@using Sandbox.UI.Construct;
```

Comment:
This using seems unnecessary

Reply:
using statements are necessary here, without it I cannot inherit WorldPanel